### DISCUSSION TITLE
clustering.lua:513: serve_cluster_listener(): failed to receive WebSocket basic_info frame: failed to receive the first 2 bytes: closed

### DISCUSSION THREAD

**Discussion by wqwwqwqwqwqwqwq** [Kong/kong#8250] [Help]
I'm using Kong 2.4.1 hybird model.When I start a Dp node ,the CP error.log print this:

2021/12/31 10:06:06 [error] 43587#0: *275158926 [lua] clustering.lua:513: serve_cluster_listener(): failed to receive WebSocket basic_info frame: failed to receive the first 2 bytes: closed, client: XXXXX, server: kong_cluster_listener, request: "GET /v1/outlet?node_id=XXXX&node_hostname=kong-dp01&node_version=2.4.1 HTTP/1.1", host: "XXXX:8005"
2021/12/31 10:06:06 [error] 43587#0: *275158926 lua entry thread aborted: runtime error: /usr/local/share/lua/5.1/kong/clustering.lua:514: attempt to call method 'close' (a nil value)
stack traceback:
coroutine 0:
        /usr/local/share/lua/5.1/kong/clustering.lua: in function 'serve_cluster_listener'
        content_by_lua(nginx-kong.conf:97):2: in main chunk, client: XXXX server: kong_cluster_listener, request: "GET /v1/outlet?node_id=XXX&node_hostname=kong-dp01&node_version=2.4.1 HTTP/1.1", host: "XXX:8005".

Please help

**Comment by dndx** [comment]
> I'm using Kong 2.4.1 hybird model.When I start a Dp node ,the CP error.log print this:
> 
> 2021/12/31 10:06:06 [error] 43587#0: *275158926 [lua] clustering.lua:513: serve_cluster_listener(): failed to receive WebSocket basic_info frame: failed to receive the first 2 bytes: closed, client: XXXXX, server: kong_cluster_listener, request: "GET /v1/outlet?node_id=XXXX&node_hostname=kong-dp01&node_version=2.4.1 HTTP/1.1", host: "XXXX:8005" 2021/12/31 10:06:06 [error] 43587#0: *275158926 lua entry thread aborted: runtime error: /usr/local/share/lua/5.1/kong/clustering.lua:514: attempt to call method 'close' (a nil value) stack traceback: coroutine 0: /usr/local/share/lua/5.1/kong/clustering.lua: in function 'serve_cluster_listener' content_by_lua(nginx-kong.conf:97):2: in main chunk, client: XXXX server: kong_cluster_listener, request: "GET /v1/outlet?node_id=XXX&node_hostname=kong-dp01&node_version=2.4.1 HTTP/1.1", host: "XXX:8005".
> 
> Please help

Which version of Kong CP are you running? Is it also 2.4.1 (same as DP)?
